internal/quickstart/show_sdk_instructions.go

```diff
@@ -20,18 +19,24 @@ const (
 	viewportHeight = 30
 )
 
+type envKeys struct {
```

Comment:
I keep getting tripped up by this name since I expect it to be a slice of the key value of multiple environments, not the various keys that are defined on an environment. What about calling it `environment` like we do with the `flag` type where it's a subset of the full data for an environment that we could add to later?

Reply:
fair enough! updated